CHANGELOG.rst

```diff
@@ -1,6 +1,16 @@
 Changelog
 =========
 
+1.7.1 (2022-07-25)
```

Comment:
Should this branch be cut from main instead? Looks like this is cut from release branch. If we merge this, then this would go in too right?